ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/KsqlTestingTool.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.test.tools;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.confluent.ksql.function.TestFunctionRegistry;
+import io.confluent.ksql.json.JsonMapper;
+import io.confluent.ksql.test.model.QttTestFile;
+import io.confluent.ksql.test.model.TestCaseNode;
+import io.confluent.ksql.test.tools.command.TestOptions;
+import io.confluent.ksql.util.Pair;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class KsqlTestingTool {
+
+  private KsqlTestingTool() {
+
+  }
+
+  private static final ObjectMapper OBJECT_MAPPER = JsonMapper.INSTANCE.mapper;
+
+  private static int totalNumberOfTests;
+
+  public static void main(final String[] args) throws IOException {
+    loadAndRunTests(args);
+  }
+
+  static void loadAndRunTests(final String[] args) {
+    totalNumberOfTests = 0;
+    final List<String> passedTests = new ArrayList<>();
+    final List<Pair<String, String>> failedTests = new ArrayList<>();
+    try {
+
+      final TestOptions testOptions = TestOptions.parse(args);
+
+      if (testOptions == null) {
+        System.err.println("Invalid parameter.");
```

Comment:
nit: these two lines are printed even when the `-h` or `--help` flag is explicitly passed.

```
Victoria-Xias-MBP13:ksql victoria.xia$ ./bin/ksql-testing-tool --help
NAME
        testingtool - KSQL testing tool

SYNOPSIS
        testingtool [--] <test-file>

OPTIONS
        --
            This option can be used to separate command-line options from the
            list of arguments (useful when arguments might be mistaken for
            command-line options)

        <test-file>
            A JSON file containing the test configurations including statements
            and test data.

            This option may occur a maximum of 1 times


Invalid parameter.
Usage: ksql-testing-tool /path/to/the/test/file.json
```

Reply:
Removed the last two lines.